pkg/cmd/openshift-controller-manager/cmd.go

```
@@ -74,8 +77,12 @@ func NewOpenShiftControllerManagerCommand(name, basename string, out, errout io.
 	flags := cmd.Flags()
 	// This command only supports reading from config
 	flags.StringVar(&options.ConfigFilePath, "config", options.ConfigFilePath, "Location of the master configuration file to run from.")
-	cmd.MarkFlagFilename("config", "yaml", "yml")
-	cmd.MarkFlagRequired("config")
+	if err := cmd.MarkFlagFilename("config", "yaml", "yml"); err != nil {
```

I think in these two cases you can ignore the error, we never verify it.